dandischema/digests/dandietag.py

```diff
@@ -0,0 +1,202 @@
+# Derived from <https://github.com/girder/django-s3-file-field/blob/master/
+# s3_file_field/_multipart.py>, copyright Kitware, Inc. <kitware@kitware.com>
+# under the Apache 2.0 license
+
+from dataclasses import dataclass
+from hashlib import md5
+import math
+import os
+from typing import Iterator, List, NamedTuple, Optional, Union
+
+
+def mb(bytes_size: int) -> int:
+    return bytes_size * 2 ** 20
+
+
+def gb(bytes_size: int) -> int:
+    return bytes_size * 2 ** 30
+
+
+def tb(bytes_size: int) -> int:
+    return bytes_size * 2 ** 40
```

Comment:
these are mebi, gibi, and tebi - bytes. perhaps its a good time to consolidate these options in relation to aws.